src/components/Sidebar/TwitchSidebar/TwitchSidebar.tsx

```diff
@@ -0,0 +1,47 @@
+import { EuiAvatar, EuiTitle } from '@elastic/eui';
+
+import styles from './TwitchSidebar.styles';
+export interface TwitchSidebarProps {
+  streams: Array<string>;
+}
+
+export interface StreamItemProps {
+  stream: string;
+  onlineStatus: string;
+}
+
+function capitalize(s) {
+  return s && s[0].toUpperCase() + s.slice(1);
```

Comment:
If you are only using this for the avatar, wouldn't t be easier to only return the first character as upper case? The rest isn't getting used and pretty sure twitch won't allow names with spaces

Reply:
word